api/src/test/java/org/motechproject/nms/api/osgi/UserControllerBundleIT.java

```diff
@@ -712,6 +722,7 @@ public void testGetUserNotInWhitelistedByState() throws IOException, Interrupted
                 ADMIN_USERNAME, ADMIN_PASSWORD));
     }
 
+    @Ignore // TODO: Renable once #119 is merged
     @Test
     public void testGetUserNotInWhitelistedByLanguageLocationCode() throws IOException, InterruptedException {
```

Comment:
Nitpick: the name of this test is not grammatically correct -- testGetUserNotInWhitelistByLanguageLocationCode?